nautobot/extras/api/serializers.py

```diff
@@ -890,7 +890,7 @@ class Meta:
         ]
 
 
-class RelationshipAssociationSerializer(serializers.ModelSerializer):
+class RelationshipAssociationSerializer(ValidatedModelSerializer):
```

Comment:
Hmmm. Looking at this file, there are a *lot* of serializers that are using `serializers.ModelSerializer` instead of Nautobot's `ValidatedModelSerializer` or even `BaseModelSerializer` as their base class. That seems almost certainly incorrect to me and a likely source of other bugs. It might be worth doing some exploratory testing here and opening a housekeeping/bug issue for testing/fixing up the following serializers as well:

```
% git grep serializers.ModelSerializer 
nautobot/dcim/api/serializers.py:class CableTerminationSerializer(serializers.ModelSerializer):
nautobot/dcim/api/serializers.py:class TracedCableSerializer(StatusModelSerializerMixin, serializers.ModelSerializer):
nautobot/dcim/api/serializers.py:class CablePathSerializer(serializers.ModelSerializer):
nautobot/extras/api/nested_serializers.py:class NestedJobSerializer(serializers.ModelSerializer):
nautobot/extras/api/nested_serializers.py:class NestedJobLogEntrySerializer(serializers.ModelSerializer):
nautobot/extras/api/nested_serializers.py:class NestedJobResultSerializer(serializers.ModelSerializer):
nautobot/extras/api/nested_serializers.py:class NestedScheduledJobSerializer(serializers.ModelSerializer):
nautobot/extras/api/serializers.py:class ContentTypeSerializer(serializers.ModelSerializer):
nautobot/extras/api/serializers.py:class JobResultSerializer(serializers.ModelSerializer):
nautobot/extras/api/serializers.py:class ScheduledJobSerializer(serializers.ModelSerializer):
nautobot/extras/api/serializers.py:class JobLogEntrySerializer(serializers.ModelSerializer):
nautobot/extras/api/serializers.py:class ObjectChangeSerializer(serializers.ModelSerializer):
nautobot/extras/api/serializers.py:class RelationshipSerializer(serializers.ModelSerializer):
nautobot/extras/api/serializers.py:class RelationshipAssociationSerializer(serializers.ModelSerializer):
```